### DISCUSSION TITLE
How to generate .pb.go files?

### DISCUSSION THREAD

**Discussion by srijan55** [cortexproject/cortex#4448] [Q&A] [answered]
Silly question, but I am trying to make a contribution which involves changing a .proto file. I am continuously running into protoc compilation issues related to imports in the .proto files. Is there a standard method which generates all the go code from proto files. "go build" doesn't seem to do so. (PS: I am building on windows)

**Comment by pstibrany** [comment, marked as answer]
Typically we use `make protos` to do this. This step uses build image running under Docker. You may want to fetch (instead of building) the build image by running `make fetch-build-image` first.
